hwdb.d/60-keyboard.hwdb

```diff
@@ -637,11 +637,16 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPProBook11G1:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP:pnHPZBookStudioG4:pvr*
  KEYBOARD_KEY_f8=wlan                                   # Wireless HW switch button
 
-# HP Folio 1040g2
+# HP EliteBook Folio 1040 G2
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPEliteBookFolio1040G2:pvr*
  KEYBOARD_KEY_d8=!f23                                   # touchpad off
  KEYBOARD_KEY_d9=!f22                                   # touchpad on
 
+# HP EliteBook Folio G1
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP:pnHPEliteBookFolioG1:pvr*
+  KEYBOARD_KEY_64=calendar
+  KEYBOARD_KEY_81=micmute
```

Comment:
Please use single space of indentation here. (Udev accepts any whitespace, but we want to be consistent in the files we ship.)

Reply:
Done